upstream.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-file]
+directory = range-v3-0.11.0
+source_url = https://github.com/ericniebler/range-v3/archive/0.11.0.zip
+source_filename = range-v3-0.11.0.zip
+source_hash = 269e6a5ec8ac9967d5a052e6e312caff5e9f739b149feb402d13719af967b2d6
```

Comment:
Needs https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section

Note: upstream does not provide pkg-config files, but they do provide range-v3-config.cmake, confirming the name "range-v3" for the provided dependency.

Reply:
Done.